dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -58,6 +58,8 @@
     # skipping fidelity estimation due to
     # https://github.com/quantumlib/Cirq/issues/3502
     "examples/*fidelity*",
+    # Also skipping stabilizer code testing.
```

Comment:
This should say why we are skipping.

Reply:
Done. I took the liberty of reusing the same issues, since the solution would likely be the same. I put a note in there.